lib/std/c.zig

```diff
@@ -210,6 +210,23 @@ pub const ARCH = switch (native_os) {
     .linux => linux.ARCH,
     else => void,
 };
+
+// For use with posix.timerfd_create()
+// Actually, the parameter for the timerfd_create() function is in integer,
+// which means that the developer has to figure out which value is appropriate.
+// To make this easier and, above all, safer, because an incorrect value leads
+// to a panic, an enum is introduced which only allows the values
+// that actually work.
+pub const CLOCK_ID = clock_id;
+pub const clock_id = switch (native_os) {
+    .linux, .freebsd => enum(u32) {
+        REALTIME = 0,
+        MONOTONIC = 1,
+        _,
+    },
+    else => void,
```

Comment:
```suggestion
    else => clockid_t,
```

When we don't know better, just fall back to `clockid_t` as before. Otherwise this represents a compile error regression on affected platforms.

Reply:
Good point, but the line numbers from your suggestion are outdated. I think I have to open a new PR?